examples/llava/llava.cpp

```diff
@@ -0,0 +1,139 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <vector>
+
+#include "clip.h"
+#include "llava-utils.h"
+#include "common.h"
+#include "llama.h"
+
+
+int main(int argc, char ** argv) {
+    ggml_time_init();
+
+    gpt_params params;
```

Comment:
as gg mentions in his tweet, it performs better with greedy sampling / low temperature.
setting temp to 0.1 by default should be good.